packages/pn-personagiuridica-webapp/src/components/Deleghe/AcceptDelegationModal.tsx

```diff
@@ -143,18 +140,18 @@ const AcceptDelegationModal: React.FC<Props> = ({
     );
   }
   return (
-    <Dialog
+    <PnDialog
```

Comment:
Checks the comments left on CodeModal about distance between Title and Content, and Content and Actions